lib/services/swagger-api-service.js

```diff
@@ -207,7 +210,12 @@ function swaggerFactory(
             res.status(status).send(data);
         })
         .catch(function(err) {
-            next(_processError(err));
+            if (res.locals.errorStatus) {
+                next();
```

Comment:
So it looks like the default swagger error handler fitting does the same thing: https://github.com/theganyo/swagger-node-runner/blob/master/fittings/json_error_handler.js, so ignore my comment.